piece_of_cake.py

```diff
@@ -0,0 +1,16 @@
+# get the prices of the ingredients ,the amount need and the optional - which is the ingredients we don't need
+# and calculate the price.
+def get_recipe_price(prices, optionals=[], **ingredients):
```

Comment:
Also, please add type hints and return types to the function.

Reply:
![Capture](https://user-images.githubusercontent.com/13467749/159872711-f95d06c2-1ca2-4cc6-a5f9-d10f2723a9e6.JPG)
It should all look like this